chore: remove multi-transaction handling from wallet services

saledjenic opened the pull request:
- Simplified transaction management by removing multi-transaction ID from `SendTransactionWithSignature` and related methods.
- Deprecated endpoints removed `WatchTransactionByChainID`, `ProceedWithTransactionsSignatures` and `GetMultiTransactions`.
- Updated transaction handling in various services to streamline the process.
- Cleaned up related tests to reflect these changes.

saledjenic committed:
chore: remove multi-transaction handling from wallet services

- Simplified transaction management by removing multi-transaction ID from `SendTransactionWithSignature` and related methods.
- Deprecated endpoints removed `WatchTransactionByChainID`, `ProceedWithTransactionsSignatures` and `GetMultiTransactions`.
- Updated transaction handling in various services to streamline the process.
- Cleaned up related tests to reflect these changes.

status-im-auto commented:
### Jenkins Builds
<details>
<summary>Click to see older builds (56)</summary>

| :grey_question: | Commit | :hash: | Finished (UTC) | Duration | Platform | Result |
|-|-|-|-|-|-|-|
| :heavy_check_mark: | 7a7a5315 | [#1](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6936/1/) | 2025-09-17 13:49:36 | ~3 min | `macos/status-go` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-250917-134621-7a7a53-pr6936.zip) |
| :heavy_check_mark: | 7a7a5315 | [#1](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6936/1/) | 2025-09-17 13:49:57 | ~3 min | `linux/status-go` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-250917-134617-7a7a53-pr6936.zip) |
| :heavy_multiplication_x: | 7a7a5315 | [#1](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6936/1/) | 2025-09-17 13:52:03 | ~5 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6936/1//consoleText) |
| :heavy_check_mark: | 7a7a5315 | [#1](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6936/1/) | 2025-09-17 13:53:09 | ~6 min | `windows/status-go` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-250917-134628-7a7a53-pr6936.zip) |
| :heavy_check_mark: | 7a7a5315 | [#1](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/nwaku/job/PR-6936/1/) | 2025-09-17 13:57:07 | ~10 min | `linux/nwaku` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/nwaku-250917-134618-7a7a53-pr6936.zip) |
| :heavy_multiplication_x: | 7a7a5315 | [#1](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6936/1/) | 2025-09-17 13:57:53 | ~11 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6936/1//consoleText) |
| | | | | | | |
| :heavy_check_mark: | 24b56c62 | [#2](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6936/2/) | 2025-09-17 14:08:39 | ~3 min | `linux/status-go` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-250917-140518-24b56c-pr6936.zip) |
| :heavy_check_mark: | 24b56c62 | [#2](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6936/2/) | 2025-09-17 14:08:47 | ~3 min | `macos/status-go` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-250917-140523-24b56c-pr6936.zip) |
| :heavy_check_mark: | 24b56c62 | [#2](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6936/2/) | 2025-09-17 14:12:10 | ~6 min | `windows/status-go` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-250917-140530-24b56c-pr6936.zip) |
| :heavy_check_mark: | 24b56c62 | [#2](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/nwaku/job/PR-6936/2/) | 2025-09-17 14:15:14 | ~9 min | `linux/nwaku` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/nwaku-250917-140521-24b56c-pr6936.zip) |
| :heavy_multiplication_x: | 24b56c62 | [#2](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6936/2/) | 2025-09-17 14:17:43 | ~12 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6936/2//consoleText) |
| :heavy_check_mark: | 24b56c62 | [#2](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6936/2/) | 2025-09-17 14:31:48 | ~26 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6936/2//consoleText) |
| :heavy_multiplication_x: | 24b56c62 | [#3](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6936/3/) | 2025-09-17 14:44:43 | ~11 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6936/3//consoleText) |
| | | | | | | |
| :heavy_check_mark: | 8909ad33 | [#3](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6936/3/) | 2025-09-17 15:47:20 | ~3 min | `linux/status-go` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-250917-154402-8909ad-pr6936.zip) |
| :heavy_check_mark: | 8909ad33 | [#3](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6936/3/) | 2025-09-17 15:47:34 | ~3 min | `macos/status-go` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-250917-154407-8909ad-pr6936.zip) |
| :heavy_check_mark: | 8909ad33 | [#3](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6936/3/) | 2025-09-17 15:51:41 | ~7 min | `windows/status-go` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-250917-154413-8909ad-pr6936.zip) |
| :heavy_check_mark: | 8909ad33 | [#3](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/nwaku/job/PR-6936/3/) | 2025-09-17 15:53:55 | ~9 min | `linux/nwaku` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/nwaku-250917-154404-8909ad-pr6936.zip) |
| :heavy_multiplication_x: | 8909ad33 | [#4](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6936/4/) | 2025-09-17 15:55:45 | ~11 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6936/4//consoleText) |
| :heavy_check_mark: | 8909ad33 | [#3](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6936/3/) | 2025-09-17 16:10:17 | ~26 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6936/3//consoleText) |
| | | | | | | |
| :heavy_check_mark: | 4f561f4f | [#4](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6936/4/) | 2025-09-17 19:30:20 | ~3 min | `linux/status-go` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-250917-192705-4f561f-pr6936.zip) |
| :heavy_check_mark: | 4f561f4f | [#4](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6936/4/) | 2025-09-17 19:30:31 | ~3 min | `macos/status-go` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-250917-192709-4f561f-pr6936.zip) |
| :heavy_multiplication_x: | 4f561f4f | [#4](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6936/4/) | 2025-09-17 19:32:30 | ~5 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6936/4//consoleText) |
| :heavy_check_mark: | 4f561f4f | [#4](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6936/4/) | 2025-09-17 19:34:08 | ~6 min | `windows/status-go` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-250917-192716-4f561f-pr6936.zip) |
| :heavy_check_mark: | 4f561f4f | [#5](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6936/5/) | 2025-09-17 19:36:56 | ~9 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6936/5//consoleText) |
| :heavy_check_mark: | 4f561f4f | [#4](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/nwaku/job/PR-6936/4/) | 2025-09-17 19:37:58 | ~10 min | `linux/nwaku` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/nwaku-250917-192706-4f561f-pr6936.zip) |
| | | | | | | |
| :heavy_check_mark: | 5c6e65fb | [#5](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6936/5/) | 2025-09-17 19:57:57 | ~3 min | `linux/status-go` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-250917-195439-5c6e65-pr6936.zip) |
| :heavy_check_mark: | 5c6e65fb | [#5](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6936/5/) | 2025-09-17 19:58:03 | ~3 min | `macos/status-go` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-250917-195443-5c6e65-pr6936.zip) |
| :heavy_check_mark: | 5c6e65fb | [#5](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6936/5/) | 2025-09-17 20:00:40 | ~5 min | `windows/status-go` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-250917-195448-5c6e65-pr6936.zip) |
| :heavy_check_mark: | 5c6e65fb | [#6](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6936/6/) | 2025-09-17 20:04:18 | ~9 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6936/6//consoleText) |
| :heavy_check_mark: | 5c6e65fb | [#5](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/nwaku/job/PR-6936/5/) | 2025-09-17 20:04:36 | ~9 min | `linux/nwaku` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/nwaku-250917-195440-5c6e65-pr6936.zip) |
| :heavy_check_mark: | 5c6e65fb | [#5](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6936/5/) | 2025-09-17 20:20:39 | ~25 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6936/5//consoleText) |
| | | | | | | |
| :heavy_check_mark: | 28b71fd3 | [#6](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6936/6/) | 2025-09-18 06:36:53 | ~3 min | `linux/status-go` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-250918-063336-28b71f-pr6936.zip) |
| :heavy_check_mark: | 28b71fd3 | [#6](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6936/6/) | 2025-09-18 06:37:06 | ~3 min | `macos/status-go` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-250918-063341-28b71f-pr6936.zip) |
| :heavy_check_mark: | 28b71fd3 | [#7](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6936/7/) | 2025-09-18 06:40:36 | ~3 min | `linux/status-go` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-250918-063700-28b71f-pr6936.zip) |
| :heavy_check_mark: | 28b71fd3 | [#7](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6936/7/) | 2025-09-18 06:40:37 | ~3 min | `macos/status-go` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-250918-063711-28b71f-pr6936.zip) |
| :heavy_check_mark: | 28b71fd3 | [#6](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6936/6/) | 2025-09-18 06:40:41 | ~6 min | `windows/status-go` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-250918-063348-28b71f-pr6936.zip) |
| :heavy_check_mark: | 28b71fd3 | [#7](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6936/7/) | 2025-09-18 06:43:11 | ~9 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6936/7//consoleText) |
| :heavy_check_mark: | 28b71fd3 | [#6](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/nwaku/job/PR-6936/6/) | 2025-09-18 06:43:57 | ~10 min | `linux/nwaku` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/nwaku-250918-063338-28b71f-pr6936.zip) |
| :heavy_check_mark: | 28b71fd3 | [#7](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6936/7/) | 2025-09-18 06:47:06 | ~6 min | `windows/status-go` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-250918-064044-28b71f-pr6936.zip) |
| :heavy_check_mark: | 28b71fd3 | [#8](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6936/8/) | 2025-09-18 06:52:38 | ~9 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6936/8//consoleText) |
| :heavy_check_mark: | 28b71fd3 | [#7](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/nwaku/job/PR-6936/7/) | 2025-09-18 06:54:07 | ~10 min | `linux/nwaku` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/nwaku-250918-064401-28b71f-pr6936.zip) |
| :heavy_check_mark: | 28b71fd3 | [#6](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6936/6/) | 2025-09-18 06:59:38 | ~25 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6936/6//consoleText) |
| :heavy_check_mark: | 28b71fd3 | [#7](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6936/7/) | 2025-09-18 07:25:20 | ~25 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6936/7//consoleText) |
| | | | | | | |
| :heavy_check_mark: | ced8419c | [#8](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6936/8/) | 2025-09-18 08:42:21 | ~3 min | `linux/status-go` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-250918-083834-ced841-pr6936.zip) |
| :heavy_check_mark: | ced8419c | [#8](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6936/8/) | 2025-09-18 08:42:51 | ~4 min | `macos/status-go` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-250918-083838-ced841-pr6936.zip) |
| :heavy_check_mark: | ced8419c | [#8](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6936/8/) | 2025-09-18 08:46:40 | ~7 min | `windows/status-go` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-250918-083844-ced841-pr6936.zip) |
| :heavy_multiplication_x: | ced8419c | [#9](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6936/9/) | 2025-09-18 08:49:18 | ~10 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6936/9//consoleText) |
| :heavy_check_mark: | ced8419c | [#8](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/nwaku/job/PR-6936/8/) | 2025-09-18 08:49:34 | ~10 min | `linux/nwaku` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/nwaku-250918-083835-ced841-pr6936.zip) |
| :heavy_check_mark: | ced8419c | [#8](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6936/8/) | 2025-09-18 09:05:27 | ~26 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6936/8//consoleText) |
| :heavy_check_mark: | ced8419c | [#10](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6936/10/) | 2025-09-18 09:17:25 | ~8 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6936/10//consoleText) |
| | | | | | | |
| :heavy_check_mark: | 2967f8a6 | [#9](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6936/9/) | 2025-09-18 09:32:46 | ~4 min | `macos/status-go` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-250918-092759-2967f8-pr6936.zip) |
| :heavy_check_mark: | 2967f8a6 | [#9](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6936/9/) | 2025-09-18 09:34:06 | ~6 min | `windows/status-go` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-250918-092805-2967f8-pr6936.zip) |
| :heavy_check_mark: | 2967f8a6 | [#9](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6936/9/) | 2025-09-18 09:36:07 | ~8 min | `linux/status-go` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-250918-092755-2967f8-pr6936.zip) |
| :heavy_check_mark: | 2967f8a6 | [#9](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/nwaku/job/PR-6936/9/) | 2025-09-18 09:44:22 | ~16 min | `linux/nwaku` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/nwaku-250918-092756-2967f8-pr6936.zip) |
| :heavy_check_mark: | 2967f8a6 | [#11](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6936/11/) | 2025-09-18 09:45:42 | ~17 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6936/11//consoleText) |
| :heavy_check_mark: | 2967f8a6 | [#9](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6936/9/) | 2025-09-18 10:01:42 | ~33 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6936/9//consoleText) |
</details>

| :grey_question: | Commit | :hash: | Finished (UTC) | Duration | Platform | Result |
|-|-|-|-|-|-|-|
| :heavy_check_mark: | 9dc24dac | [#10](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6936/10/) | 2025-09-18 13:18:36 | ~3 min | `macos/status-go` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-250918-131514-9dc24d-pr6936.zip) |
| :heavy_check_mark: | 9dc24dac | [#10](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6936/10/) | 2025-09-18 13:23:06 | ~7 min | `windows/status-go` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-250918-131519-9dc24d-pr6936.zip) |
| :heavy_check_mark: | 9dc24dac | [#10](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6936/10/) | 2025-09-18 13:40:22 | ~25 min | `linux/status-go` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-250918-131509-9dc24d-pr6936.zip) |
| :heavy_check_mark: | 9dc24dac | [#10](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/nwaku/job/PR-6936/10/) | 2025-09-18 13:49:57 | ~34 min | `linux/nwaku` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/nwaku-250918-131511-9dc24d-pr6936.zip) |
| :heavy_check_mark: | 9dc24dac | [#12](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6936/12/) | 2025-09-18 13:51:27 | ~36 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6936/12//consoleText) |
| :heavy_check_mark: | 9dc24dac | [#10](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6936/10/) | 2025-09-18 14:08:54 | ~53 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6936/10//consoleText) |
| | | | | | | |
| :heavy_check_mark: | c68dd95c | [#11](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6936/11/) | 2025-09-22 13:44:37 | ~3 min | `linux/status-go` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-250922-134103-c68dd9-pr6936.zip) |
| :heavy_check_mark: | c68dd95c | [#11](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6936/11/) | 2025-09-22 13:44:39 | ~3 min | `macos/status-go` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-250922-134107-c68dd9-pr6936.zip) |
| :heavy_check_mark: | c68dd95c | [#11](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6936/11/) | 2025-09-22 13:47:56 | ~6 min | `windows/status-go` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-250922-134113-c68dd9-pr6936.zip) |
| :heavy_check_mark: | c68dd95c | [#11](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/nwaku/job/PR-6936/11/) | 2025-09-22 13:52:56 | ~11 min | `linux/nwaku` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/nwaku-250922-134104-c68dd9-pr6936.zip) |
| :heavy_check_mark: | c68dd95c | [#13](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6936/13/) | 2025-09-22 13:53:47 | ~12 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6936/13//consoleText) |
| :heavy_check_mark: | c68dd95c | [#11](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6936/11/) | 2025-09-22 14:10:24 | ~29 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6936/11//consoleText) |

saledjenic committed:
chore: simplify transaction handling by removing multi-transaction ID

- Updated multiple services and tests to eliminate multi-transaction ID from `SendTransactionWithSignature` and related methods.
- Adjusted transaction management logic across the code base.
- Cleaned up related tests.

saledjenic committed:
chore: remove multi-transaction ID references and clean up code

- Eliminated multi-transaction ID from various structures and functions across the wallet service.
- Updated related queries and database interactions to reflect the removal of multi-transaction handling.
- Cleaned up related tests.

saledjenic committed:
chore: remove unused methods from path processors

- Eliminated `Send` and `BuildTransaction` methods.
- Updated related tests to reflect the removal of these methods.
- Codebase cleaned up accordingly.

dlipicar reviewed (commented):
hmm looks good in principle, I'd just be very surprised if this didn't break the app's activity history with no status-desktop changes.
Also would like @vkjr to take a look.

saledjenic commented:
> I'd just be very surprised if this didn't break the app's activity history with no status-desktop changes

@dlipicar there is a desktop PR that includes all these code removal-related PRs:
- https://github.com/status-im/status-desktop/pull/18856

I've tested a bit, did some tx, and it was working, but waiting for @anastasiyaig confirmation before merging.
